### DISCUSSION TITLE
GitHub Copilot Issue

### DISCUSSION THREAD

**Discussion by AliRazaSaqib** [community/community#17210] [Copilot Conversations] [answered]
I am trying to connect the GitHub Copilot extension in vs code but it will show an error message. I am also sending you an invitation to Github to join the extension with GitHub but I will say you receive an email, it will take more than 1 month maybe but not receive any email.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
